bucket/chromium.json

```diff
@@ -1,37 +1,49 @@
 {
-    "version": "75.0.3770.142-r652427",
-    "description": "An open-source browser project that aims to build a safer, faster, and more stable way for all users to experience the web.",
+    "##": "Check chromium.woolyss.com for different versions of Chromium releases.",
+    "version": "76.0.3809.100-r665002",
+    "description": "Open-source browser project that aims to build a safer, faster, and more stable way for all users to experience the web.",
     "license": "BSD-3-Clause",
     "homepage": "https://www.chromium.org",
-    "checkver": {
-        "url": "https://chromium.woolyss.com/api/v3/?os=windows&bit=64&type=stable-codecs-sync&out=string",
-        "re": "v([\\d.]+-r(?:\\d+))-win64"
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/macchrome/winchrome/releases/download/v76.0.3809.100-r665002-Win64/Chromium-76.0.3809.100-x64.7z",
+            "hash": "sha1:67fda51228441ec59bdbfa4de046f4ea4f4d551b",
+            "extract_dir": "Chromium-76.0.3809.100-x64"
+        },
+        "32bit": {
+            "url": "https://github.com/macchrome/winchrome/releases/download/v76.0.3809.100-r665002-Win64/Chromium-76.0.3809.100-x86.7z",
+            "hash": "sha1:b3a9c7bfaf220ba2f0868047e1043b74246a9c96",
+            "extract_dir": "Chromium-76.0.3809.100-x86"
+        }
     },
     "bin": "chrome.exe",
-    "extract_dir": "chrome-win32",
     "shortcuts": [
         [
             "chrome.exe",
             "Chromium"
         ]
     ],
-    "architecture": {
-        "64bit": {
-            "url": "https://github.com/henrypp/chromium/releases/download/v75.0.3770.142-r652427-win64/chromium-sync.zip",
-            "hash": "9b70c145471e8bcd4d717e0c6d736e1cb94c9710f18a5638f5cc1175418d9e0b"
-        },
-        "32bit": {
-            "url": "https://github.com/henrypp/chromium/releases/download/v75.0.3770.142-r652427-win32/chromium-sync.zip",
-            "hash": "bc33cf72da162dc90374957ed2d4e9e8026b51caca1290122a4e0cba218d04cb"
-        }
+    "checkver": {
+        "github": "https://github.com/macchrome/winchrome",
+        "regex": "v((?<main>[\\d.]+)-r\\d+)-Win64"
```

Comment:
I think you do not need capture group for this.

Keep it all as version and then try `$matchHead` instead of `$matchMain`

Reply:
I tried `$matchHead`, but **head** only matches `76.0.3809`, not `76.0.3809.100`.

Reply:
```suggestion
        "regex": "v([\\d.]+-r\\d+)-Win64"
```